09-mapping.Rmd

```diff
@@ -399,20 +425,26 @@ The preceding code chunk demonstrates key features of faceted maps created with
 - Shapes that do not have a facet variable are repeated (the countries in `world` in this case).
 - The `by` argument which varies depending on a variable (`year` in this case).
 - `nrow`/`ncol` setting specifying the number of rows and columns that facets should be arranged into.
-- The `free.coords`argument specifying if each map has its own bounding box.
+- The `free.coords`-parameter specifying if each map has its own bounding box.
 
-In addition to their utility for showing changing spatial relationships, facet maps are also useful as the foundation for animated maps (see \@ref(animated-maps)).
+In addition to their utility for showing changing spatial relationships, faceted maps are also useful as the foundation for animated maps (see \@ref(animated-maps)).
 
 ### Inset maps
 
-The role of spatial visualizations is to provide an information in the most accessible form. 
-This is often done with the use of inset maps.
+An inset map is a smaller map rendered within or next to the main map. 
+<!-- I would say inset maps are usually used for the opposite. This is we have a specific area, say a mountain somewhere in Peru, and the inset map shows were in Peru the mountain is located, see Figure 3 in https://journal.r-project.org/archive/2017/RJ-2017-067/RJ-2017-067.pdf
+and
+http://wiki.gis.com/wiki/index.php/Inset_Map
+So maybe it would be an idea to focus on the NZ Alps raster map as the main map and use NZ within the inset map.
+Maybe this section is also a bit too long. How about shortening it, and maybe use the US example as a challenge in the exercise section.
+-->
+Inset maps usually cover an area with densely located phenomena that cannot be clearly visible at the original map scale.
```

Comment:
I would say inset maps are usually used for the opposite. This is we have a specific area, say a mountain somewhere in Peru, and the inset map shows were in Peru the mountain is located, see Figure 3 in https://journal.r-project.org/archive/2017/RJ-2017-067/RJ-2017-067.pdf
and
http://wiki.gis.com/wiki/index.php/Inset_Map

So maybe it would be an idea to focus on the NZ Alps raster map as the main map and use NZ within the inset map.

Secondly, this section maybe is also a bit too long. How about shortening it, and maybe use the US example as a challenge in the exercise section.

Reply:
Fair point - could you try to make the changes needed to make this point in the example + prose?